### ISSUE TITLE
minimum_width parameter not respected when output_format is not i3bar

### ISSUE THREAD

**Issue by msuperdock** [i3/i3status#258]
It seems that the minimum_width parameter in the i3status config file is only respected when the output_format is i3bar. It would be great if the other output formats supported a minimum width as well.

Below is a minimal example config file. On running i3status, I expected to see "one          | two", with the appropriate number of spaces after "one". Instead I see "one | two", indicating that the minimum width is ignored.

I have confirmed that minimum_width is also ignored when output_format is equal to "term" and "dzen2". When output_format is "i3bar", the result is a JSON object which includes min_width as a field, so I'm guessing i3bar is handling the minimum-width logic. My suggestion is for this logic to be handled by i3status for the other output formats. (I am using dzen2 in particular.)

```
general {
        output_format = "none"
}

order += "load"
order += "battery 0"

load {
        format = "one"
        min_width = "                    "
}

battery 0 {
        format = "two"
}
```

**Comment by Airblader** [comment]
I don't see how i3status can fulfill the contract `min_width` has, because it also allows numeric values in pixels. I think we should rather clarify that this is only supported for i3bar.

**Comment by msuperdock** [comment]
Good point--my thought was that it would still make sense for string values to be accepted. But I think it would also make sense just to clarify that it is not supported. (For me in particular it would have helped if it were clarified in sections 7 and 8 of the manpage: https://i3wm.org/i3status/manpage.html#_using_i3status_with_dzen2)

**Comment by Airblader** [comment]
PRs to fix the docs are welcome. :-)
